angular-chess/src/app/components/chess-field/chess-field.component.scss

```diff
@@ -0,0 +1,43 @@
+.field {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 10vh;
+  width: 10vh;
+}
+
+.field--dark {
+  background-image: url(../../../../images/board_field_dark.png);
+}
+
+.field--light {
+  background-image: url(../../../../images/board_field_light.png);
+}
+
+.piece {
+  height: 100%;
+  width: 100%;
+  font-size: 9.3vh;
+  font-family: "Chess Motif";
+  text-align: center;
+}
+
+.piece-WHITE {
+  color: white;
+}
+
+.piece-BLACK {
+  color: black;
+}
+
+.color-RED {
+  background: rgba(red, 0.4);
+}
+
+.color-GREEN {
+  background: rgba(green, 0.4);
+}
+
+.color-YELLOW {
+  background: rgba(yellow, 0.4);
+}
```

Comment:
When we need yellow?

Reply:
It is planned to add a feature to highlight different squares. It's useful if you want to analyse a position. A yellow field could mean that a field is attacked but currently sufficiently defended (e.g. 2 attackers vs 2 defenders). I will create a ticket for it.

Reply:
Here is the ticket: https://github.com/RubikNube/angular-chess/issues/37

Reply:
unused -> removed